backend/fms_core/resources/transfer.py

```diff
@@ -60,116 +59,139 @@ class Meta:
+            obj.individual = self.transferred_from.individual
+            obj.concentration = self.transferred_from.concentration
+        else:
+            self.manuallyExclude.extend(["name", "sample_kind", "collection_site", "individual", "concentration"]) 
+
+        # This section creates the container if needed
+        get_normalized_str(data, "Destination Parent Container Barcode")
+
+        parent = None
+        destination_parent_container_barcode = get_normalized_str(data, "Destination Parent Container Barcode")
+
+        if destination_parent_container_barcode:
+            try:
+                parent = Container.objects.get(barcode=destination_parent_container_barcode)
+            except Container.DoesNotExist:
+                errors["destination parent container"] = ValidationError(["Destination parent container does not exist."], code="invalid")
+
+        shared_container_info =  dict(
+            barcode=get_normalized_str(data, "Destination Container Barcode")
+        )
+        if parent:
+            shared_container_info['location'] = parent
+        if get_normalized_str(data, "Destination Container Name"):
+            shared_container_info['name'] = get_normalized_str(data, "Destination Container Name")
+        if get_normalized_str(data, "Destination Container Kind"):
+            shared_container_info['kind'] = get_normalized_str(data, "Destination Container Kind")
+        if get_normalized_str(data, "Destination Parent Container Coord"):
+            shared_container_info['coordinates'] = get_normalized_str(data, "Destination Parent Container Coord")
+        try:
+            obj.container, container_created = Container.objects.get_or_create(
                 **shared_container_info,
                 defaults={'comment': f"Automatically generated via transfer template import on "
                         f"{datetime.utcnow().isoformat()}Z"},
             )
+        except Exception as e:
+            errors["container"] = ValidationError([f"Could not create destination container. Destination Container Barcode already exists and related fields do not match: "
+                                                   f"[Destination Container Name, Destination Container Kind, Destination Parent Container Barcode, Destination Parent Container Coord] "], code="invalid")
+
+        try:
+            # Create a process for the current transfer
+            self.process = Process.objects.create(protocol=Protocol.objects.get(name="Transfer"),
+                                                  comment="Sample Transfer (imported from template)")
+            self.process_sample = ProcessSample.objects.create(process=self.process,
+                                                               source_sample=self.transferred_from,
+                                                               execution_date=obj.creation_date,
+                                                               volume_used=obj.volume,
```

Comment:
obj.volume is confusing, I thought by mistake you had put the volume of the obj, so I had to go check the code where it was defined. ideally the name would not be misleading :)